cmd/terramate/cli/cloud.go

```diff
@@ -88,7 +88,7 @@ func (c *cli) disableCloudFeatures(err error) {
 func (c *cli) handleCriticalError(err error) {
 	if err != nil {
 		if c.uimode == HumanMode {
-			fatal(err)
+			fatal("Aborting", err)
```

Comment:
nitpick: for consistency with the other messages I think this should not be capitalized.